README.md

```diff
@@ -7,6 +7,8 @@ generation tools and exposes XRM-conformant managed resources for
 
 ## Getting Started
 
+This provider requires Kubernetes v1.23.x or greater. This is due to an issue when creating a large number of CRDs in previous versions.
```

Comment:
Actually, only the version with `-preview` suffix requires patched versions, `v0.4.0` is perfectly fine for example. You can see all the versions here https://github.com/crossplane-contrib/provider-jet-aws/releases/tag/v0.4.0-preview